src/hooks/useDiscordConnect.ts

```diff
@@ -0,0 +1,94 @@
+import { useCallback } from 'react'
+
+import { Web3Provider } from '@ethersproject/providers'
+
+import { Governance } from '../clients/Governance'
+import { SegmentEvent } from '../entities/Events/types'
+import { GATSBY_DISCORD_PROFILE_VERIFICATION_URL } from '../entities/User/constants'
+import { Account, AccountType } from '../entities/User/types'
+import { openUrl } from '../helpers'
+
+import useValidationSetup from './useValidationSetup'
+
+const getMessage = async () => Governance.get().getValidationMessage(Account.Discord)
+const VALIDATION_CHECK_INTERVAL = 10 * 1000 // 10 seconds
```

Comment:
This constant is used in both connect hooks. Maybe you can import it from the useValidationSetup hook?